aten/src/ATen/native/mps/OperationUtils.h

```diff
@@ -263,6 +263,8 @@ struct MPSGraphCache
 
 };
 
+bool supportsVenturaOps();
```

Comment:
I was thinking of something like [this](https://github.com/pytorch/pytorch/pull/88532/files#diff-c7bb5bafa54cf2dfc86cafb769b80f8a8dfbf3342020f11b1b3fae507b56fef2R30)

Reply:
Thank you for the suggestion. Let me move it to MPSDevice (though really its not a function of device, but rather function of an OS, which is why I wanted to make it a static method)